ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml

```diff
@@ -3898,10 +3898,13 @@ $defs:
           type: bool
           default: false
           description: Enable BFD (only considered for BGP).
-        ptp_enable:
-          type: bool
-          default: false
-          description: Enable PTP
+        ptp:
```

Comment:
this is duplicate of the existing ptp key above.

Reply:
Please elaborate. I don't understand as I don't see any duplicates in this file.

Reply:
nvm I mixed up with another data model.